x-pack/plugins/monitoring/public/components/beats/overview/__snapshots__/overview.test.js.snap

```diff
@@ -263,8 +275,7 @@ exports[`Overview that overview page shows a message if there is no beats data 1
       data-test-subj="noRecentActivityMessage"
       iconType="gear"
       size="m"
-      title="Hi there! This area is where your latest Beats activity would show
-up, but you don't seem to have any activity within the last day."
+      title="Hi there! This area is where your latest Beats activity would show up, but you don't seem to have any activity within the last day."
```

Comment:
Shouldn't this be translated too?

Reply:
This is a snapshot that is generated automatically during running tests.  Such label was translated here https://github.com/elastic/kibana/pull/24976/files#diff-786e975d1cfabc594efdeec009061671R78.